### PR TITLE
tests: run tests on opensuse leap 15.0 instead of 42.3

### PR DESCRIPTION
**Pull request by sergiocazzolato**
This is using the new image already created on google.

### CODE REVIEW

File: spread.yaml [hunk 1]
```diff
@@ -84,6 +84,8 @@ backends:
                 workers: 4
```
Comment (mvo5) [inline spread.yaml]: Should we remove  - opensuse-42.3-64: from above or may this come back eventually?

Comment (zyga) [reply spread.yaml]: I would keep it until its EOL later this year.

**Comment by zyga** [comment]
It would be good to run this with `--debug` and check what exactly failed here:
```
+ echo 'systemctl reports the system is in degraded mode'
systemctl reports the system is in degraded mode
+ systemctl --failed
  UNIT                           LOAD   ACTIVE SUB    DESCRIPTION          
â systemd-vconsole-setup.service loaded failed failed Setup Virtual Console
```

File: tests/main/interfaces-desktop-host-fonts/task.yaml [hunk 2]
```diff
@@ -16,12 +16,7 @@ restore: |
     rm -f /usr/lib/fontconfig/cache/cache.txt
 
 execute: |
-    #shellcheck source=tests/lib/pkgdb.sh
-    . "$TESTSLIB"/pkgdb.sh
-    distro_install_package fontconfig
-
```
Comment (zyga) [inline tests/main/interfaces-desktop-host-fonts/task.yaml]: I'm curious about this part. There are changes with additions of fontconfig above and here removal of the installation of the fontconfig package. What is this about?

Comment (zyga) [reply tests/main/interfaces-desktop-host-fonts/task.yaml]: Ahh, I'm blind. Sorry :)

**Pull request merged**